### ISSUE TITLE
Unexpected result encountered for X:-Y case

### ISSUE THREAD

**Issue by jasonyukr** [theryangeary/choose#42]
Please check the test result below.

```
~% echo "1/2/3/4/5" | choose -f '/' -o '/' 1:-1
2/3/4/5
~% echo "1/2/3/4/5" | choose -f '/' -o '/' 1:-2
2/3/4
~% echo "1/2/3/4/5" | choose -f '/' -o '/' 1:-3
2/3
~% echo "1/2/3/4/5" | choose -f '/' -o '/' 1:-4

~%

```
I expected "2" from the last run. But nothing returned.

**Comment by theryangeary** [comment]
Thanks for the report. This definitely looks like unexpected behavior. I'll take a look.

**Comment by theryangeary** [comment]
This has been resolved by #49, and is available in `develop` now. Look out for the next release (hopefully soon) which will make it available via precompiled binaries!

**Comment by jasonyukr** [comment]
Great. Thanks for the fix.
